pyspelling/__init__.py

```diff
@@ -196,6 +198,13 @@ def _walk_src(self, targets, flags, pipeline):
                     except Exception as e:
                         err = self.get_error(e)
                         yield [filters.SourceText('', f, '', '', err)]
+        if not found_something:
+            raise ValueError(
+                'None of the source targets {} from the configuration'
```

Comment:
Maybe for easier reading?

> None of the source targets from the configuration match any files: {}.

Might be better than breaking the sentence with a long list of patterns.

Or maybe even:

> None of the source targets from the configuration match any files:
> \- pattern1
> \- pattern2

You probably could leave out the surrounding quotes then maybe :man_shrugging: .

Reply:
Yes great idea, really like the look of the second one - will do.